##[error]Unexpected end of JSON input. failed to resolve path=/etc/polkit-1: failed to resolve link

When I use actions scan, I get an error message：



[0040] DEBUG layer metadata: index=11 digest=sha256:b59b5ad61c206eae23387c367f586d37a9f91d79cccbf8c6bb1a188db6cbb487 mediaType=application/vnd.docker.image.rootfs.diff.tar.gzip from-lib=stereoscope
[0043]  INFO identified distro: centos 7 from-lib=syft
[0043]  INFO cataloging image from-lib=syft
[0043] DEBUG cataloger 'ruby-gemspec-cataloger' discovered '6' packages from-lib=syft
[0051] DEBUG cataloger 'python-package-cataloger' discovered '20' packages from-lib=syft
[0051] DEBUG cataloger 'javascript-package-cataloger' discovered '0' packages from-lib=syft
[0051] DEBUG cataloger 'dpkgdb-cataloger' discovered '0' packages from-lib=syft
[0052] DEBUG cataloger 'java-cataloger' discovered '18' packages from-lib=syft
[0052] DEBUG cataloger 'apkdb-cataloger' discovered '0' packages from-lib=syft
[0052] DEBUG cataloger 'go-cataloger' discovered '0' packages from-lib=syft
[0052] ERROR failed to catalog: 1 error occurred:
	* unable to catalog rpmdb package=/var/lib/rpm/Packages: failed to resolve path=/etc/polkit-1: failed to resolve link from img (ref=[85779] /etc/polkit-1): failed to resolve link metadata ([85779] /etc/polkit-1): could not find file


##[error]Unexpected end of JSON input


 code :


Dockerfile

```
FROM centos:centos7
RUN yum -y update
RUN yum -y update
RUN rpm -iUvh http://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/e/epel-release-7-13.noarch.rpm
RUN yum install -y expect redis-server ansible java-1.7.0-openjdk wget unzip openssh-clients gcc-c++ make ruby ruby-rdoc ruby-devel jq
RUN yum install -y gcc-c++ make
RUN yum install -y golang


ADD . ./
# Create root2 account
RUN	useradd -d /root2 -m -s /bin/bash -U root2
ADD ./startup.sh  /root2/startup.sh
ADD ./main.go  /root2/main.go
RUN chown -R root2:root2 /root2/startup.sh
RUN chmod 777 /root2/*
USER	root2
WORKDIR /root2

CMD ["/root2/startup.sh"]
```

startup.sh

```bash
#!/bin/bash

#mkdir /tmp/pro-itg
#chmod 755 /tmp/pro-itg
ls -l
go version
go run main.go
```

main.go
 
```go 
/*
@File: main.go
@Thinks
*/
package main

import (
	"fmt"
)

func main() {
	fmt.Println("hello go ")
}
```

--- a/index.js
+++ b/index.js
@@ -4,7 +4,7 @@ const { exec } = require('@actions/exec');
 const fs = require('fs');
 
 const grypeBinary = 'grype'
-const grypeVersion = '0.5.0'
+const grypeVersion = '0.6.1'
 
 // sarif code
 function convert_severity_to_acs_level(input_severity, severity_cutoff_param) {

